cves/kernel/CVE-2016-9685.yml

```diff
@@ -75,7 +78,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: [1400368, 1396941]
```

Comment:
Not sure these bug ids are on https://bugzilla.kernel.org? Do you know where these refer to? If you change them to links that would be fine too.